client/components/admin/AdminTeams/imports/AdminTeamModalGeneral.jsx

```diff
@@ -50,7 +50,9 @@ class AdminTeamModalGeneral extends Component {
                         <strong> Division:</strong> {team.division}
                     </Grid.Column>
                     <Grid.Column>
-                        <strong> Looking for members?</strong> {team.lookingForMembers ? "Yes" : "No"}
+                        <strike>
+                            <strong> Looking for members?</strong> n/a
```

Comment:
Is there a reason for this to be crossed out and N/A instead of just removing it?

Reply:
I left it kind of as a reminder for myself to put it back later. But I suppose if I revisit I'll probably re-examine a lot of fields so that may not be needed. can delete if you feel that'd be better

Reply:
Okay! It's okay as is, we're the only ones really seeing it anyway.